api/tests/test_signals.py

```diff
@@ -0,0 +1,10 @@
+import pytest
+
+
+@pytest.mark.django_db
+def test_message_audit_update_triggers_message_reindex(
+    ratom_message_audit, mock_core_registry
```

Comment:
This can be removed now, right?

Reply:
the `mock_core_registry`

Reply:
isn't it being used on line 10?

Reply:
I think Colin's change makes it apply everywhere. 

> Autouse fixtures (xUnit setup on steroids)
> The class-level transact fixture is marked with autouse=true which implies that all test >methods in the class will use this fixture without a need to state it in the test function >signature or with a class-level usefixtures decorator.

Reply:
``mock_core_registry`` is used everywhere, but it needs to be imported here to check the values passed to the mock object

Reply:
Maybe it's still useful to keep here so we can assert that it is called?